src/react-apps/applications/runtime/src/shared/resources/attachments/map/mapAttachmentsSagas.ts

```diff
@@ -5,19 +5,29 @@ import { IRuntimeState } from '../../../../types';
 import { mapAttachmentListToAttachments } from '../../../../utils/attachment';
 import AttachmentDispatcher from '../attachmentActions';
 import * as AttachmentActionsTypes from '../attachmentActionTypes';
-import { IData } from './../../../../../../shared/src/types';
+import { IData, IInstance } from './../../../../../../shared/src/types';
+import { returnCurrentTaskData } from './../../../../../../shared/src/utils/applicationMetaDataUtils';
```

Comment:
Maybe `getCurrentTaskData` is a better name?

Reply:
Renamed.